lib/apiary/command/styleguide.rb

```diff
@@ -68,7 +94,7 @@ def validate
       end
 
       begin
-        load
+        load(true, true, true)
```

Comment:
This call looks really weird IMO - can't we use named arguments? Or pass an array of things we want?

Reply:
fixed